reconcile/templating/rendering.py

```diff
@@ -0,0 +1,106 @@
+from abc import ABC, abstractmethod
+from typing import Any, Optional
+
+from jinja2.sandbox import SandboxedEnvironment
+from pydantic import BaseModel
+from ruamel import yaml
+
+from reconcile.gql_definitions.templating.templates import TemplateV1
+from reconcile.utils.jsonpath import parse_jsonpath
+
+
+class Renderer(ABC):
+    template: TemplateV1
+    data: "TemplateData"
+
+    def __init__(self, template: TemplateV1, data: "TemplateData"):
```

Comment:
Please move the `TemplateData` class above this class and you can specify `data: TemplateData` without quotes